.env

```diff
@@ -34,3 +34,7 @@ PROMPT_STORYGPT_SERVICE_URL=http://prompt-storygpt:8127/respond
 STORYGPT_SERVICE_URL=http://storygpt:8126/respond
 SENTENCE_RANKER_SERVICE_URL=http://sentence-ranker:8128/respond
 FILE_SERVER_URL=http://files:3000
+TERMINUSDB_SERVER_URL=http://terminusdb-server:6363
+TERMINUSDB_SERVER_TEAM=admin
+TERMINUSDB_SERVER_DB=user_knowledge_db
```

Comment:
@dilyararimovna 
As we removed remote connection, shall we just fix these values to its default in test.py and, later, in user_kg annotator? Or, it's still better to give users the option to edit them in case they change DB settings like team, password, and db name?